### ISSUE TITLE
Mime.getTypesFromExtension don't support uppercase

### ISSUE THREAD

**Issue by redDwarf03** [niteshsh4rma/mime_dart#7]
If a file contains an extension in uppercase(for example: image.JPG), the method ``Mime.getTypesFromExtension()`` get nothing.

```
    final fileDecoded = File(file.path).readAsBytesSync();
    final typeMime = Mime.getTypesFromExtension(
      path.extension(file.path).replaceAll('.', ''),
    )![0];
```

Could you manage this case please ?
Thx

**Comment by niteshsh4rma** [comment]
Thanks @redDwarf03, for coming up with this, I will add an update to the library and release a new version soon.

**Comment by niteshsh4rma** [comment]
@redDwarf03  Fixed in Release 3.0.0, closing this
